packages/webui/src/client/ui/RundownView.tsx

```diff
@@ -311,9 +311,10 @@ const TimingDisplay = withTranslation()(
 						<TimeOfDay />
 						{rundownPlaylist.currentPartInfo && (
 							<span className="timing-clock current-remaining">
-								<CurrentPartRemaining
+								<CurrentPartOrSegmentRemaining
 									currentPartInstanceId={rundownPlaylist.currentPartInfo.partInstanceId}
 									heavyClassName="overtime"
+									useSegmentTime={true}
```

Comment:
I think this will change the current behavior? No change in behavior was announced in an RFC. Also, `useSegmentTime=true` seems to cause `displayTimecode = this.props.timingDurations.remainingBudgetOnCurrentSegment`, and `remainingBudgetOnCurrentSegment` will be undefined, unless one is using `segment.budgetDuration`, causing the countdown to go blank?